package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "playwright_repo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "test.js",
+  "type":"commonjs",
+  "scripts": {},
```

Comment:
Its better to have scripts here, at least basic ones. Not everybody use VScode :)

Reply:
added scripts:
 "scripts":  {
    "test": "playwright test",
    "build": "webpack",
    "start": "node server.js"
  },